techniques/system/server-roles/1.0/init-check.st

```diff
@@ -67,10 +57,6 @@ bundle agent generic_process_check_process(binary, initscript, name, force_resta
 		"@@server-roles@@result_error@@&TRACKINGKEY&@@Check ${name} process@@None@@${g.execRun}##${g.uuid}@#The ${name} process couldn't be restarted"
 			ifvarclass => "process_restart_error_${canoname}";
 
-    policy_server.!root_server.!process_exception_on_relay::
-
-		"@@server-roles@@result_success@@&TRACKINGKEY&@@Check ${name} process@@None@@${g.execRun}##${g.uuid}@#The ${name} process do not need to be checked on relay servers. Skipping...";
```

Comment:
This exact same report is now done in component-check.st when checking a process is not necessary.